src/content/configuration/output.md

```diff
@@ -422,20 +453,14 @@ module.exports = _entry_return_;
 require("MyLibrary").doSomething();
 ```
 
-T> Wondering the difference between CommonJS and CommonJS2? Check [this](https://github.com/webpack/webpack/issues/1114) out (they are pretty much the same).
+Note that `output.library` is omitted, thus it is not required for this particular `output.libraryType`.
 
+T> Wondering the difference between CommonJS and CommonJS2? Check [this](https://github.com/webpack/webpack/issues/1114) out (while they are similar, there are some subtle differences that are not usually relevant in the context of webpack).
```

Comment:
@metatoaster Better would be "Wondering what the difference between CommonJS and CommonJS2 is?

Reply:
@metatoaster And maybe "Read [this issue](https://github.com/webpack/webpack/issues/1114)" instead of "Check [this](https://github.com/webpack/webpack/issues/1114) out"

Reply:
Again, copied verbatim from existing text, but while I am at it, as you said...